sass/components/_dialog.scss

```diff
@@ -86,7 +86,7 @@
             padding: 10px 0px 10px 12px;
         }
         &--action {
-            padding: 16px;
```

Comment:
suppose it's better to add `{
                                              display: flex;
                                              justify-items: center;
                                              padding: 16px;
                                          }`

Reply:
@mahdiyeh-fs , without `display: flex; justify-items: center; ` result will be the same. You can check. if I will use `padding: 16px; `instead of `padding: 16px 16px 8px 8px; , then design will be changed. I'm not sure that’s it good practice to change layout where it’s not required.

Reply:
changing the padding to `0 16px;` will fix that.